src/Evolution/Systems/Ccz4/Tags.hpp

```diff
@@ -49,6 +49,67 @@ template <size_t Dim, typename Frame, typename DataType>
 using InverseConformalMetric =
     gr::Tags::Conformal<gr::Tags::InverseSpatialMetric<Dim, Frame, DataType>,
                         -2>;
+
+/*!
+ * \brief The natural log of the lapse
+ */
+template <typename DataType>
+struct LogLapse : db::SimpleTag {
+  using type = Scalar<DataType>;
+};
+
+/*!
+ * \brief Auxiliary variable which is analytically the spatial derivative of the
+ * natural log of the lapse
+ *
+ * \details If \f$ \alpha \f$ is the lapse, then we define
+ * \f$A_i = \partial_i ln(\alpha) = \frac{\partial_i \alpha}{\alpha}\f$.
+ */
+template <size_t Dim, typename Frame, typename DataType>
+struct DerivLogLapse : db::SimpleTag {
+  using type = tnsr::i<DataType, Dim, Frame>;
+};
+
+/*!
+ * \brief Auxiliary variable which is analytically the spatial derivative of the
+ * shift
+ */
+template <size_t Dim, typename Frame, typename DataType>
+using DerivShift = ::Tags::deriv<gr::Tags::Shift<Dim, Frame, DataType>,
```

Comment:
Will do!

Just for my own understanding, I feel that I understand that `B_k{}^i` will not numerically equal to the derivative of the shift, but what is the practical difference between making a new tag for this numeric value vs. using the `deriv` prefix tag on the `Shift` tag to represent it? Is this just a spectre convention used to designate that something does not equal the analytic value (i.e. so the reader/user knows), or is there a computational difference between the two options?

Reply:
it's sort of convention, we use `deriv` for numerically differentiate quantities. The other is that in this case `B` is its own variable, there's just a constraint that it be equal to the derivative of the shift. I have a less strong opinion against naming tags like this with short names. I'd suggest `FieldB` as the name because `DerivShift` will likely be confused as the actual derivative of the shift

Reply:
I see, I will make the name change

Reply:
Well whichever way you want to name the tags, please make them consistent within the `Ccz4` namespace. I'll let the two of you decide the naming scheme

Reply:
@nilsdeppe, in that case I will defer to you

Reply:
What you have now is good :) 

There will be places where you use `B` and places where you use the derivative of the shift once constraint damping is added in. That's fine, this is how every second-order system that's rewritten in first-order form works. The "consistency" depends on maintaining hyperbolicity of the evolution system

Reply:
Sorry, based on what @nilsleiffischer said, it sounds like he would like us to choose between one of the two conventions within the `Ccz4::Tags` namespace:
(1) me changing `FieldB` back to `DerivShift`, or
(2) me changing `DerivLogLapse` to `FieldA`, `HalfDerivConformalMetric` to `FieldD`, and `DerivLogConformalFactor` to `FieldP`

I could be misinterpreting, but I think this is what was meant by being consistent, since currently, `FieldB` is a thing, but `FieldA`, `FieldD`, and `FieldP` are not. Nils Fischer, let me know if I'm misunderstanding

Reply:
ah, I see. I would prefer `FieldA`, `FieldD` etc. so that they match the paper better and so people don't confuse them as the actual numerical derivative of the lapse, shift, etc.

Reply:
Sounds good, will do